src/onegov/landsgemeinde/forms/votum.py

```diff
@@ -117,13 +118,33 @@ def next_number(self):
         query = query.limit(1)
         return (query.scalar() or 0) + 1
 
+    def populate_person_choices(self):
+        people = PersonCollection(self.request.session).query()
+        people_choices = [(
+            (
+                f'{p.first_name} {p.last_name}, {p.function}, '
+                f'{p.political_party}, {p.location_code_city}, '
+                f'{p.picture_url}',
+                f'{p.first_name} ' + ', '.join(filter(None, [
+                    p.last_name,
+                    p.function,
+                    p.political_party,
+                    p.location_code_city])))
+        ) for p in people]
+        people_choices.insert(0, (', , , ', '...'))
```

Comment:
Is this the correct number of `,`? The generated choices appear to use 4, rather than just 3. The same would apply to the current `default`/`choices` parameters on the field.

Reply:
You're right, i's not the correct number. I'll fix that.